src/util/mod.rs

```diff
@@ -122,3 +123,10 @@ impl<T> HandyRwLock<T> for RwLock<T> {
         self.read().unwrap()
     }
 }
+
+
+pub fn make_std_tcp_conn<A: ToSocketAddrs>(addr: A) -> Result<TcpStream, Error> {
```

Comment:
Actually `make_tcp_stream` is shorter than `make_std_tcp_conn`. And it's named `TcpStream` in std lib.